task-scheduler.dart

```diff
@@ -17,8 +17,11 @@ class TaskScheduler {
 
 
   // View all tasks
-  void viewAllTask() {
-
+  void viewAll() {
+    for (Task task in taskList) {
+      print(
+          "Title: ${task.title}, Description: ${task.description}, Due Date: ${task.dueDate}, Completed: ${task.status}");
+    }
   }

```

Comment:
Nit: You could have done just "print(task) or  used print(task.toString()" just for consistency sake.

Reply:
Nit: You have to print all the properties on new lines